tests/h/services/search_index/service_test.py

```diff
@@ -55,10 +55,6 @@ def test_it_also_adds_the_thread_root(
 
         add_annotation.assert_has_calls([call(reply_annotation), call(root_annotation)])
 
-    @pytest.fixture(autouse=True)
-    def storage(self, patch):
-        return patch("h.services.search_index.service.storage")
```

Comment:
This was patching a module-level name but was a class-scoped fixture. Our style is to patch module-level names with module-scoped autouse fixtures, so I've moved this to module scope. It means this fixture is usable by all tests that might be calling code that accesses `storage`, and also means that there aren't any tests that _should_ be patching `storage` but are accidentally using the real `storage`